float64 ropeLength
float64 springConstant
float64 ropeWeight
string frame
float64 x
float64 y
float64 z
---